docker/use-guide.md

```diff
@@ -0,0 +1,61 @@
+Docker for Kylin is convenient for users to quickly try Kylin, as well as for developers to debug and verify code modifications.
+
+In the image, including:
+* Jdk 1.8
+* Hadoop 2.7.0
+* Hive 1.2.1
+* Hbase 1.1.2
+* Spark 2.3.1
+* Zookeeper 3.4.6
+* Mysql
+* Maven 3.6.1
+
+## Build image
+Execute the following command to build image
+```
+sh build_image.sh
+```
+This operation may last for several tens of minutes due to the download of the installation package.
+
+## Launch container
+After image built, execute
+```
+sh run_container.sh
+```
+to launch container which will automatically start the following services:
+* Mysql
+* NameNode
+* DataNode
+* ResourceManager
+* NodeManager
+* HMaster
+
+Then execute command [docker exec](https://docs.docker.com/engine/reference/commandline/exec/) to run command in then running Kylin container.
+
+## For user to try Kylin
+All Kylin service dependences are ready, just need to download Kylin release binary from http://kylin.apache.org/download/ and start Kylin service.
+
+For example:
+```
+cd /home/admin
+wget https://archive.apache.org/dist/kylin/apache-kylin-3.0.0-alpha2/apache-kylin-3.0.0-alpha2-bin-hbase1x.tar.gz
+tar -zxvf apache-kylin-3.0.0-alpha2-bin-hbase1x.tar.gz
+cd apache-kylin-3.0.0-alpha2-bin-hbase1x
+bin/kylin.sh start
+```
+
+## For developer to debug and verify
+When building image, the latest kylin source code will be copied to `/home/admin/kylin_sourcecode`.
+
+Execute the following commands to do package:
+```
+cd /home/admin/kylin_sourcecode
+build/script/package.sh
+```
+After the package is complete, use the package in `/home/admin/kylin_sourcecode/dist` to start the Kylin service.
+
+You can also run CI inside the container:
```

Comment:
If Integration test can not pass currently, please remove this, thank you.

Reply:
fixed